css/pages/_content.scss

```diff
@@ -149,13 +163,13 @@
             background: $white;
         }
 
-        &:nth-child(odd) {
+        &::nth-child(odd) {
```

Comment:
This is an invalid variant for pseudo-classes, double colons are only valid for pseudo-elements.
Please check it everywhere.